ptero_workflow/implementation/models/methods/shell_command.py

```diff
@@ -28,7 +30,7 @@ class ShellCommand(Method):
     }
 
     VALID_CALLBACK_TYPES = Method.VALID_CALLBACK_TYPES.union(
-            ['begun', 'error', 'execute', 'failure', 'success'])
+            ['execute', 'running', 'succeeded', 'errored', 'failed'])
```

Comment:
Should the statuses imported from `ptero_common.statuses` be used here, instead of these strings?